src/civis/response.py

```diff
@@ -400,6 +401,29 @@ def __next__(self):
         return next(self._iter)
 
 
+class ListResponse(list):
+    __slots__ = ("headers",)
+
+    def __init__(self, responses, headers=None):
+        super().__init__(responses)
+        self.headers = headers
+
+    def json(self, snake_case=True):
```

Comment:
Defining the `json` method since the `job_logs` function [expects it](https://github.com/civisanalytics/civis-python/pull/518/files#diff-b20815f075736d92e1673daecb2417bfa94d58d3b911841b016f557d20c05fe7R230).